### PR TITLE
New KB

### PR DESCRIPTION
**Pull request by mark-turpin**
Load Balance Additional Networks

### CODE REVIEW

**Comment by chrislittle** [comment]
Mark,
you need a line inserted before your images otherwise (per the preview) your image just follows the text and it looks bad (just hit preview in github).  you may want to use numbers for the steps (even the end items like "After you have completed adding networks the RNAT page will display the networks..") not bullets too.  

i'd love for you to add a link to this KB in the LB comparison matrix additional information area too

File: Network/import-pfx-dedicated-load-balancers.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+* HA Dedicated Load Balancers have been deployed to your account
+* Must have an Admin login to the Netscaler
+* An exported PFX certificate from a Windows machine with password
+
+### Steps
+
+1. Log into your dedicated load balancer RNAT IP (typically ends with .101)
+2. Expand Traffic Management > SSL and click on Import PKCS#12 (under the Tools section)
+
+![LB Menu](../images/load-balance-with-windows-certificates-1.png)
```
Comment (nathanyoung) [inline Network/import-pfx-dedicated-load-balancers.md]: broken image?

File: Network/import-pfx-dedicated-load-balancers.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+3. Enter the output file name (such as wildcard.contoso.com).  Browsing is not required and you don't need to add a file extension
+4. Browse and add the PFX from Local (assumes you have the exported PFX on the system you are connecting from)
+5. Enter the PFX password
+
+![LB Menu](../images/load-balance-with-windows-certificates-2.png)
```
Comment (nathanyoung) [inline Network/import-pfx-dedicated-load-balancers.md]: broken

File: Network/import-pfx-dedicated-load-balancers.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+6. Once the certificate has been imported, you can view it in the Manage Certificates section:
+
+![LB Menu](../images/load-balance-with-windows-certificates-3.png)
```
Comment (nathanyoung) [inline Network/import-pfx-dedicated-load-balancers.md]: broken

File: Network/import-pfx-dedicated-load-balancers.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+1. Log into your dedicated load balancer RNAT IP (typically ends with .101)
+2. Expand Traffic Management > SSL and click on Import PKCS#12 (under the Tools section)
+
+![LB Menu](../images/import-pfx-dedicated-load-balancers-1.png)
+
+3. Enter the output file name (such as wildcard.contoso.com).  Browsing is not required and you don't need to add a file extension
+4. Browse and add the PFX from Local (assumes you have the exported PFX on the system you are connecting from)
+5. Enter the PFX password
+
+![LB Menu](../images/import-pfx-dedicated-load-balancers-2.png)
```
Comment (nathanyoung) [inline Network/import-pfx-dedicated-load-balancers.md]: this line needs to be indented (two spaces) for the numbered list to continue after it.

File: Network/loadbalance-additional-networks.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+### Steps
+
+1. Log into your dedicated load balancer RNAT IP (typically ends with .101)
+2. Expand System > Network and click on Routes
+3. Select the RNAT tab
+4. Select the Action drop down menu, and select Configure RNAT
+5. Add the network and subnet mask
+
+Example:  10.10.10.0 for network, 255.255.255.0 for netmask
```
Comment (nathanyoung) [inline Network/loadbalance-additional-networks.md]: indent two spaces

File: Network/loadbalance-additional-networks.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+![LB Menu](../images/loadbalance-additional-networks-1.png)
```
Comment (nathanyoung) [inline Network/loadbalance-additional-networks.md]: indent two spaces

**Pull request merged**